wandb/sdk/lib/wburls.py

```diff
@@ -27,7 +27,7 @@ def _get_urls(self) -> Dict["URLS", str]:
             cli_launch="https://wandb.me/launch",
             doc_run="https://wandb.me/run",
             doc_require="https://wandb.me/library-require",
-            doc_start_err="https://docs.wandb.ai/library/init#init-start-error",
+            doc_start_err="https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-",
```

Comment:
should we replace this one with a wandb.me link?

Reply:
Good idea. We may eventually do this, but it should be part of a large effort to replace the links in bulk.